internal/tools/pipeline/providers/reconciler/taskpolicy/provider.go

```diff
@@ -82,7 +84,10 @@ func (p *provider) resetTask(task *spec.PipelineTask, statuses ...apistructs.Pip
 	}
 	pipeline := statusPipelines[0]
 
-	beforeSuccessTask, err := p.dbClient.GetPipelineTask(pipeline.ParentTaskID)
+	if pipeline.ParentTaskID == nil {
+		return errors.New("Parent Task ID is null")
```

Comment:
please see the next few lines about `beforeSuccessTask`.

Reply:
Done.